rdrf/rdrf/utils.py

```diff
@@ -441,3 +442,164 @@ def error(text):
 
     def __repr__(self):
         return self.text
+
+
+
+class TimeStripper(object):
+    """
+    This class exists to fix an error we introduced in the migration
+    moving from Mongo to pure Django models with JSON fields ( "Modjgo" 0objects.)
+    CDE date values were converted  into iso strings including a time T substring.
+    This was done recursively for the cdes and history collections 
+    We provide forwards and backward methods to be used in a migration to fix this
+    I have used a class so we can unit test more easily.
+    """
+    
+    def __init__(self, dataset):
+        self.dataset = dataset # queryset live , lists of data records for testing
+        self.backup_data = {}
```

Comment:
I think the contents of this (what has been updated) could be written out to a file after the script finished, so we have a record of what exactly has been updated.

A bit of extra precaution doesn't hurt.